### PR TITLE
Update Version Dumping with New Config Options

### PR DESCRIPTION
**Pull request by nicegamer7**
This PR makes it so that the version info dumped contains all config options. It also corrects a typo introduced with #182 (the name of the config option).

### CODE REVIEW

File: src/main/java/com/onarandombox/MultiverseNetherPortals/MultiverseNetherPortals.java [hunk 1]
```diff
@@ -343,26 +343,37 @@ public int getProtocolVersion() {
...
+        buffer += logAndAddToPasteBinBuffer("Nether Prefix: " + this.getNetherPrefix());
+        buffer += logAndAddToPasteBinBuffer("Nether Suffix: " + this.getNetherSuffix());
+        buffer += logAndAddToPasteBinBuffer("End Prefix: " + this.getEndPrefix());
+        buffer += logAndAddToPasteBinBuffer("End Suffix: " + this.getEndSuffix());
+        buffer += logAndAddToPasteBinBuffer("Nether Links: " + this.getWorldLinks());
+        buffer += logAndAddToPasteBinBuffer("End Links: " + this.getEndWorldLinks());
+        buffer += logAndAddToPasteBinBuffer("[Multiverse-NetherPortals] Bounceback: " + this.isUsingBounceBack());
+        buffer += logAndAddToPasteBinBuffer("[Multiverse-NetherPortals] Teleport Entities: " + this.isTeleportingEntities());
+        buffer += logAndAddToPasteBinBuffer("[Multiverse-NetherPortals] Send Disabled Portal Message: " + this.isSendingDisabledPortalMessage());
+        buffer += logAndAddToPasteBinBuffer("[Multiverse-NetherPortals] Send No Destination Message: " + this.isSendingNoDestinationMessage());
```
Comment (dumptruckman) [inline src/main/java/com/onarandombox/MultiverseNetherPortals/MultiverseNetherPortals.java:355]: why are some prefixed and some not?

Comment (dumptruckman) [reply src/main/java/com/onarandombox/MultiverseNetherPortals/MultiverseNetherPortals.java:355]: I think the prefix here is wrong cause the `logAndAddToPasteBinBuffer` does that.

File: src/main/java/com/onarandombox/MultiverseNetherPortals/MultiverseNetherPortals.java [hunk 2]
```diff
@@ -343,26 +343,37 @@ public int getProtocolVersion() {
...
+                + "[Multiverse-NetherPortals] End Suffix: " + this.getEndSuffix() + System.lineSeparator()
+                + "[Multiverse-NetherPortals] Nether Links: " + this.getWorldLinks() + System.lineSeparator()
+                + "[Multiverse-NetherPortals] End Links: " + this.getEndWorldLinks() + System.lineSeparator()
+                + "[Multiverse-NetherPortals] Bounceback: " + this.isUsingBounceBack() + System.lineSeparator()
+                + "[Multiverse-NetherPortals] Teleport Entities: " + this.isTeleportingEntities() + System.lineSeparator()
+                + "[Multiverse-NetherPortals] Send Disabled Portal Message: " + this.isSendingDisabledPortalMessage() + System.lineSeparator()
+                + "[Multiverse-NetherPortals] Send No Destination Message: " + this.isSendingNoDestinationMessage() + System.lineSeparator()
+                + "[Multiverse-NetherPortals] Special Code: " + "FRN001" + System.lineSeparator();
     }
 }
```
Comment (dumptruckman) [inline src/main/java/com/onarandombox/MultiverseNetherPortals/MultiverseNetherPortals.java:379]: Where is this used?

File: src/main/java/com/onarandombox/MultiverseNetherPortals/MultiverseNetherPortals.java (hunk 1 above)
Comment (nicegamer7) [reply src/main/java/com/onarandombox/MultiverseNetherPortals/MultiverseNetherPortals.java:355]: My mistake. The method isn't used though, it's deprecated and I think we should remove it in MV 5.0.0. Do you still want me to fix it?

File: src/main/java/com/onarandombox/MultiverseNetherPortals/MultiverseNetherPortals.java (hunk 2 above)
Comment (nicegamer7) [reply src/main/java/com/onarandombox/MultiverseNetherPortals/MultiverseNetherPortals.java:379]: It's used in the MVVersionEvent listener.

File: src/main/java/com/onarandombox/MultiverseNetherPortals/MultiverseNetherPortals.java (hunk 1 above)
Comment (nicegamer7) [reply src/main/java/com/onarandombox/MultiverseNetherPortals/MultiverseNetherPortals.java]: Fixed it.

**Pull request merged**